### DISCUSSION TITLE
Carbon Web Components v2 Release Candidate 1

### DISCUSSION THREAD

**Discussion by kennylam** [carbon-design-system/carbon-for-ibm-dotcom#10609] [Show and tell]
Hello everyone! The first `@carbon/web-components` v2 release candidate has been released ([v2-next](https://www.npmjs.com/package/@carbon/web-components?activeTab=versions))! Please use this space to report any bugs, ask any questions, or just to share your thoughts and experiences using the package. We look forward to hearing from you, thanks!

**Comment by abdonrd** [comment]
Congrats to the team! 👏

**Comment by abdonrd** [comment]
About this issue:
- https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/10602

I would like to know if you will look over this soon since changing the name of properties could result in a breaking change.

**Comment by pac0rro** [comment]
Hi, team. Good work!. I was waiting for this Carbon v11 update.

Is there any documentation or full project example for this specific version?. How can I get a working example running this version?

Thanks

**Reply by pac0rro** [reply]
I got working a project with carbon web components 2.0 RC:
├── @carbon/styles@1.33.1
├── @carbon/web-components@2.0.0-rc.2
├── @ibm/plex@6.3.0
├── @intlify/unplugin-vue-i18n@0.8.2
├── @vitejs/plugin-vue@4.2.3
├── sass@1.63.6
├── vite@4.4.4
├── vue-i18n@9.2.2
├── vue-router@4.2.4
└── vue@3.3.4
...
I have some questions. To be more specific on this RC version... where can I post those questions?

Thank you

**Comment by abdonrd** [comment]
When you will merge the `cwc-v2` branch into `main`?

**Comment by reinisg** [comment]
Thanks for the new release.
I am using  "@carbon/web-components": "^2.0.1" and got one question. What are good practice syntax to include icon from icons folder, for example `@carbon/web-components/es/icons/edit/16.js` 
Want to include the Icon for <cds-icon-button> and <cds-header-global-action> elements. Can someone give advice? 

Regards, Reinis
